From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jon Hunter <jonathanh@nvidia.com>
Date: Tue, 14 Apr 2026 13:56:38 +0100
Subject: [PATCH 174/245] NVIDIA: SAUCE: hsierrrptinj: Enable by default for
 Tegra

Enable CONFIG_TEGRA_HSIERRRPTINJ by default for Tegra devices to avoid
having to enable via the defconfig.

Change-Id: Ie9333cafbcacffbc40d6a0a083f0a39c8482b034
Signed-off-by: Jon Hunter <jonathanh@nvidia.com>

diff --git a/drivers/platform/tegra/Kconfig b/drivers/platform/tegra/Kconfig
index 11157e5514bf2..3b5a66ffd7e4d 100644
--- a/drivers/platform/tegra/Kconfig
+++ b/drivers/platform/tegra/Kconfig
@@ -30,7 +30,7 @@ config TEGRA_EPL
 config TEGRA_HSIERRRPTINJ
 	bool "Enable Tegra HSI Error Report Injection client driver"
 	depends on MAILBOX && TEGRA_EPL && DEBUG_FS
-	default n
+	def_bool y if ARCH_TEGRA
 	help
 	  The tegra-hsierrrptinj driver provides a
 	  mechanism for HSI error report transmission
-- 
2.43.0

